website/docs/api/entitylinker.md

```diff
@@ -9,6 +9,12 @@ api_string_name: entity_linker
 api_trainable: true
 ---
 
+An Entity Linker component disambiguates textual mentions (tagged as named
```

Comment:
We should probably decide on either "The entity linker" or "The `EntityLinker`"

Reply:
You mean that those two variants are OK, but not the mixture "Entity Linker", right? (I agree, just want to double check you don't want to reduce it down to only one of the two options across the docs)

Reply:
Yeah, exactly, sorry if I phrased this in a confusing way!

Reply:
Ok, I think the edit I made is fine then?